### PR TITLE
Vaccine request/detail

### PR DESCRIPTION
**Pull request by rindibudiaramdhan**
# Changes
## 1. Change Status Vaccine Request Feature
 - Add change status confirm dialog
 - Validating Total products that need to be review by reviewer in Recommendation Phase
 - validating Total products that need to be review by reviewer in Final Phase
## 2. Update Vaccine Products at Recommendation Phase
- Validating Form
- If recommendation status "Not Available" or "Not Yet Fulfilled", user no need to insert recommendation product
## 3. Update Vaccine Products at Final Phase
- Validating Form
- If final status "Not Available" or "Not Yet Fulfilled", user no need to insert final product
## 4. Finalize Vaccine Request Feature

# Evidence
title: Pikobar Vaksinasi - Fitur MVP Detail Permohonan Vaksin
project: Pikobar Logistik
participants: @rindibudiaramdhan 

cc: @jabardigitalservice/jds-frontend  ❤️

### CODE REVIEW

File: src/views/requestVaccine/detail.vue [hunk 1]
```diff
@@ -25,8 +25,8 @@
               </v-list-item>
             </v-card>
             <v-card
-              v-else
-              class="mx-auto"
+              v-else-if="status > 0"
+              class="mx-auti"
```
Comment (maulanayuseph) [inline src/views/requestVaccine/detail.vue:29]: I think this typo `mx-auto` 😄

File: src/views/requestVaccine/detail.vue [hunk 2]
```diff
@@ -53,10 +53,19 @@
 
         <v-row class="mt-n5">
           <v-col cols="2" class="mr-3">
-            <span class="text-title">Status</span>
+            <span class="text-title">{{ $t('label.status') }}</span>
           </v-col>
           <v-col>
-            <span class="red--text font-weight-bold">: {{ $t('label.' + vaccineRequest.status) }}</span>
+            <span :class="(status < 2 ? 'red--text' : 'green--text text--darken-2') + ' font-weight-bold'">: {{ $t('status.' + (vaccineRequest.status || statusTarget)) }}</span>
```
Comment (maulanayuseph) [inline src/views/requestVaccine/detail.vue:59]: _To be discussed_
You can used template literals so which makes them useful for string interpolation

```
<span
  :class="`(${status} < 2) ? 'red--text' : 'green--text text--darken-2') font-weight-bold`"
>
  {{ `${$t('status.'(vaccineRequest.status || statusTarget))}` }}
</span>
```

**Review by maulanayuseph — APPROVED** [review]

**Review by adzharamrullah — APPROVED** [review]

File: src/views/requestVaccine/detail.vue (hunk 1 above)
Comment (rindibudiaramdhan) [reply src/views/requestVaccine/detail.vue:29]: thanks!

File: src/views/requestVaccine/detail.vue (hunk 2 above)
Comment (rindibudiaramdhan) [reply src/views/requestVaccine/detail.vue:59]: thanks for suggest!

**Review by naufalihsank — APPROVED** [review]

**Pull request merged**